packages/react/types/css.d.ts

```diff
@@ -9303,6 +9303,8 @@ export namespace AtRule {
     fontWeight?: FontWeight;
     src?: string;
     unicodeRange?: string;
+		sizeAdjust?: string;
+  	ascentOverride?: string;
```

Comment:
This was probably accidental, but you should try to maintain the existing indentation.

Reply:
oh, thanks i will